skripchenkoveta/hometask-5/4-3.sql

```diff
@@ -0,0 +1 @@
+SELECT CONCAT(last_name, '-', first_name) FROM actor WHERE first_name LIKE 'W%' OR last_name LIKE '%H'
```

Comment:
last_name like '%H' does not add any results, because all the surnames ends with h (lowercase). Please fix. The fix should be case insensitive, so no matter surname ends with h or H - this record should be added to the result